tinker/faculty_bios/faculty_bio_controller.py

```diff
@@ -435,6 +436,15 @@ def update_structure(self, faculty_bio_data, sdata, rform, faculty_bio_id=None):
         else:
             add_data['id'] = None
 
+        # Today's Date: (08/09/18)
+        # This hacky chunk of code is used to update the asset to include the courseleaf-user option.
+        # The courseleaf-user option is a new addition to the form and instead of having to edit and submit every single
```

Comment:
Maybe also mention why we would have to edit and submit every faculty bio. And that is submit in Cascade, not in tinker.

Reply:
And maybe mention that its a placeholder - line 448 sets the data

Reply:
fixed